src/software/simulated_tests/kickoff_friendly_play_test.cpp

```diff
@@ -14,6 +14,8 @@ class KickoffFriendlyPlayTest : public SimulatedTestFixture
 
 TEST_F(KickoffFriendlyPlayTest, test_kickoff_friendly_play)
 {
+    // TODO: ball still slows down when collides with enemy during chip
```

Comment:
Outstanding TODO. Either remove or include a ticket #.

Reply:
Whoops, it's been resolved so removed it.